### DISCUSSION TITLE
Solspace Calendar

### DISCUSSION THREAD

**Discussion by laurabennett** [verbb/navigation#406] [Ideas]
It would be great to add Calendars and Events as nodes. I'm guessing the work around is to do this with Custom?

**Comment by engram-design** [comment]
All elements are supported out of the box actually, you can enable them in the Navigation settings under Permissions.

**Reply by laurabennett** [reply]
Entries
Categories
Assets
Tags
Passive
Custom

No Calendar

**Reply by engram-design** [reply]
Looks like this is more of an issue with the Solspace Calendar not registering their Event as an element, so Craft doesn't know its a registered element (and so neither does Navigation).

Just sent through a [PR](https://github.com/solspace/craft-calendar/pull/328) to the plugin.
